apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -681,9 +697,13 @@ const (
 	// resources on the member cluster. Otherwise, we will report the difference.
 	WhenToTakeOverTypeIfNoDiff WhenToTakeOverType = "IfNoDiff"
 
-	// WhenToTakeOverTypeAlways will always apply the resource to the member cluster regardless
+	// WhenToTakeOverTypeAlways will always apply manifests to the member cluster regardless
 	// if there are differences between the resource on the hub cluster and the existing resources on the member cluster.
 	WhenToTakeOverTypeAlways WhenToTakeOverType = "Always"
+
+	// WhenToTakeOverTypeNever instructs Fleet to never apply manifests to a member cluster
+	// if there are corresponding pre-existing resources.
```

Comment:
Hi Ryan! This concerns only pre-existing resources on a individual basis, i.e., it will not stop Fleet from creating new resources. This is consistent with the other two options.

Reply:
I will update the comments to make things more clear.

Reply:
Discussed offline -> to clarify: ReportDiff has been updated to function with or without ownership; this is a code-level change that does not require API modifications. With the Never takeover option and the ReportDiff apply strategy, one can achieve the desired use case in which Fleet touches nothing but still reports configuration differences. Comments have been revised to highlight this.